distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestCatalog.java

```diff
@@ -187,6 +192,43 @@ public void testFilterPlugin() {
         }
     }
 
+    @Test
+    public void testContentDirectoryMonitor() throws Exception {
+        startFeature(true, "content-core-directorymonitor");
+        final String TMP_PREFIX = "tcdm_";
+        Path tmpDir = Files.createTempDirectory(TMP_PREFIX);
+        tmpDir.toFile().deleteOnExit();
+        Path tmpFile = Files.createTempFile(tmpDir, TMP_PREFIX, "_tmp.xml");
+        tmpFile.toFile().deleteOnExit();
+        Files.copy(this.getClass().getClassLoader().getResourceAsStream("metacard5.xml"), tmpFile,
+                StandardCopyOption.REPLACE_EXISTING);
+        LOGGER.debug("RCZ:" + tmpFile.toAbsolutePath().toString());
+
+
+        Map<String, Object> cdmProperties = new HashMap<>();
+        cdmProperties.putAll(getMetatypeDefaults("content-core-directorymonitor",
+                "ddf.content.core.directorymonitor.ContentDirectoryMonitor"));
+        cdmProperties.put("monitoredDirectoryPath", tmpDir.toString() + "/"); // Must end with /
+        cdmProperties.put("directive", "STORE_AND_PROCESS");
+        createManagedService("ddf.content.core.directorymonitor.ContentDirectoryMonitor",
+                cdmProperties);
+
+        long startTime = System.nanoTime();
+        ValidatableResponse response = executeOpenSearch("xml", "q=*SysAdmin*");
+        while (TimeUnit.NANOSECONDS.toSeconds(System.nanoTime() - startTime) < 9) {
```

Comment:
Looks like you're waiting 9 seconds right now, so just use the one minute one. The polling value doesn't really matter, you could sleep for 1 millisecond and it would still work.

9 seconds seems an arbitrary wait time, was there a reason for it?